knowhere/utils/BitsetView.h

```diff
@@ -214,7 +214,7 @@ class ConcurrentBitset {
             return x;
         };
         for (size_t i = 0; i < len; ++i) {
-            ret += __builtin_popcountl(*p_data);
```

Comment:
we are rewriting BitsetView, you can skip this new

Reply:
I see the PR https://github.com/milvus-io/knowhere/pull/125 ,  note: seems __builtin_popcountl is still used in the PR and it's a 32bit function in Windows/MinGW.

Reply:
I have tested under ubuntu, both 1UL and 1ULL occupy 8 bytes.
Since __builtin_popcountl accept 32bit in MinGW, I'll replace it with __builtin_popcountll